templates/ChoiceSchema.stencil

```diff
@@ -0,0 +1,6 @@
+
+public enum {{ choice.language.default.name }}: {{ choice.choiceType.language.default.name }}, Codable {
```

Comment:
I think it will make more sense if the filename here reflects what the end-result Swift code is. So this would be `Enumeration.stencil` instead of `ChoiceSchema.stencil`. As we get into the more complex stuff, a single file will be composed of several schemas.

Open to ideas, but I think when the stencil describes the entire file, it should have a name that roughly matches the destination. For example, `Enumerations.swift` will be generated from the `Enumerations.stencil` and `StorageBlobClient.swift` will be generated from `ServiceClient.stencil`. 

For partial templates that are only used to compose others (like enum choices), I would recommend something like `EnumerationChoiceSnippet.stencil` and `HeaderSnippet.stencil`.  

Thoughts?